FS.test_exfat integration-test failure

Version: 2.8.3

Trying to run `src/tests/integration-test FS.test_exfat` I get the following failure:
```
# src/tests/integration-test FS.test_exfat
Testing installed system binaries
daemon path: /usr/lib/udisks2/udisksd
Set up test device: r/w: /dev/sda, r/o: /dev/sr1
test_exfat (__main__.FS)
fs: exFAT ... [cli] FUSE exfat 1.3.0
FUSE exfat 1.3.0
WARN: volume was not unmounted cleanly.
FUSE exfat 1.3.0
FUSE exfat 1.3.0
WARN: volume was not unmounted cleanly.
[ud] FAIL

======================================================================
FAIL: test_exfat (__main__.FS)
fs: exFAT
----------------------------------------------------------------------
Traceback (most recent call last):
  File "src/tests/integration-test", line 694, in test_exfat
    self._do_fs_check('exfat')
  File "src/tests/integration-test", line 859, in _do_fs_check
    self._do_udisks_check(fs_type)
  File "src/tests/integration-test", line 1074, in _do_udisks_check
    self.fail('Mounting read-only device didn\'t fail with \'rw\' option.')
AssertionError: Mounting read-only device didn't fail with 'rw' option.

----------------------------------------------------------------------
Ran 1 test in 49.258s

FAILED (failures=1)
```
```
# apt-cache policy exfat-fuse exfat-utils
exfat-fuse:
  Installed: 1.3.0-1
  Candidate: 1.3.0-1
  Version table:
 *** 1.3.0-1 500
        500 http://deb.debian.org/debian sid/main amd64 Packages
        100 /var/lib/dpkg/status
exfat-utils:
  Installed: 1.3.0-1
  Candidate: 1.3.0-1
  Version table:
 *** 1.3.0-1 500
        500 http://deb.debian.org/debian sid/main amd64 Packages
        100 /var/lib/dpkg/status
```

Test system is Debian sid.

--- a/src/tests/integration-test
+++ b/src/tests/integration-test
@@ -935,8 +935,8 @@ class FS(UDisksTestCase):
 
         self.assertProperty(fs, 'mount-points', [mount_a])
 
-        if fs_type != 'ntfs':
-            # ntfs-3g does not support multiple mounts
+        if fs_type not in ('ntfs', 'exfat'):
+            # ntfs-3g and exfat don't support multiple mounts
             subprocess.check_call([mount_prog, self.device, mount_b])
             self.assertProperty(fs, 'mount-points', {mount_a, mount_b})
             subprocess.call(['umount', mount_b])

--- a/src/tests/integration-test
+++ b/src/tests/integration-test
@@ -1065,7 +1065,8 @@ class FS(UDisksTestCase):
         # this is known-broken for reiserfs and xfs right now:
         # https://github.com/karelzak/util-linux/issues/17
         # https://github.com/karelzak/util-linux/issues/18
-        if fs_type not in ['reiserfs', 'xfs']:
+        # exfat can be mounted with '-o rw' even if read-only
+        if fs_type not in ['reiserfs', 'xfs', 'exfat']:
             # the scsi_debug CD drive content is the same as for the HD drive, but
             # udev does not know about this; so give it a nudge to re-probe
             subprocess.call(['partprobe', self.cd_device])

